xmantissa/port.py

```diff
@@ -275,6 +275,68 @@ def listen(self):
 registerAttributeCopyingUpgrader(SSLPort, 1, 2)
 
 
+
+class EndpointPort(PortMixin, Item):
```

Comment:
`StringEndpointPort`, perhaps, since endpoints generated without strings are still endpoints, and the distinguishing feature here is a string?